test/streamed_request_test.dart

```diff
@@ -24,4 +24,9 @@ void main() {
       expect(() => request.contentLength = 10, throwsStateError);
     });
   });
+  group('#invalid method', () {
```

Comment:
Same naming as in other test: `"#method"` and `"must be token"`.

Reply:
fixed. Thanks